cmd/contour/servecontext.go

```diff
@@ -165,6 +173,8 @@ func newServeContext() *serveContext {
 			Name:          "leader-elect",
 		},
 		UseExperimentalServiceAPITypes: false,
+		envoyServiceName:               "envoy",
+		envoyServiceNamespace:          "projectcontour",
```

Comment:
We should consume the downward env variable for this so we don't keep the hard coded default.

Reply:
Good catch, I'll set that up.